tests/PHPUnit/Integration/Db/Schema/TiDbTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Db\Schema;
+
+use Piwik\Config\DatabaseConfig;
+use Piwik\Db;
+use Piwik\Tests\Framework\TestCase\IntegrationTestCase;
+
+class TiDbTest extends IntegrationTestCase
```

Comment:
```suggestion
class TidbTest extends IntegrationTestCase
```

Should also receive a matching filename change, but that keeps it identical to the actual schema class case.